src/main/java/graphql/nadel/engine/NadelExecutionStrategy.java

```diff
@@ -211,6 +223,19 @@ public NadelExecutionStrategy(List<Service> services,
         return resultNodes;
     }
 
+    private RootExecutionResultNode getSkippedServiceCallResult(NadelContext nadelContext, ExecutionStepInfo esi, ExecutionContext newExecutionContext, TransformationMetadata.NormalizedFieldAndError topLevelFieldError) {
```

Comment:
Change the type of `topLevelFieldError` to just `GraphQLError`. It's weird that the value type holds both a field AND error, when we only want the error. It just provides a type safe way to ensure the right value is passed in.